config.json

```diff
@@ -2,14 +2,14 @@
   "language": "C",
   "slug": "c",
   "active": true,
-  "blurb": "C is a small, general-purpose, imperative programming language with a static type system, scopes, and structures. It's typically used as an alternative to assembly programming, such as in operating systems.",
```

Comment:
Same as for the other PR applies: This was modified by `configlet create`. Let me know if I should revert it or put it into a different PR.